spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/abstractnn/AbstractModule.scala

```diff
@@ -372,8 +372,48 @@ abstract class AbstractModule[A <: Activity: ClassTag, B <: Activity: ClassTag,
     this
   }
 
-  def cloneModule(): AbstractModule[A, B, T] = {
-    SerializationUtils.clone(this)
+  def cloneModule(deepCopy : Boolean = true): AbstractModule[A, B, T] = {
```

Comment:
Don't remove the previous implementation; instead, add a new method `clone` with the new implementation.